features/detached_auto_attach.feature

```diff
@@ -0,0 +1,33 @@
+@uses.config.contract_token
+Feature: Attached cloud does not detach when auto-attaching after manually attaching
+
+    @series.all
+    @uses.config.machine_type.aws.generic
+    @uses.config.machine_type.azure.generic
+    @uses.config.machine_type.gcp.generic
+    Scenario Outline: No detaching on manually attached machine on all clouds
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua refresh` with sudo
+        Then I will see the following on stdout:
+            """
+            Successfully processed your ua configuration.
+            Successfully refreshed your subscription.
+            """
+        When I run `ua auto-attach` with sudo
+        Then stderr matches regexp:
+        """
+        Skipping attach: Instance '[0-9a-z\-]+' is already attached.
+        """
+        Then stdout does not match regexp:
```

Comment:
Let's just add a step here where we check `status` to verify that the machine is still attached. My only concern is that we could change the `This machine is now detached` message in the future and then the test will not be fully testing that scenario anymore